src/dmd/backend/outbuf.d

```diff
@@ -72,9 +72,19 @@ struct Outbuffer
         return ret;
     }
 
-    // Make sure we have at least `nbyte` available for writting
+    // Make sure we have at least `nbytes` available for writing
+    // Reserve nbytes in buffer.
```

Comment:
```suggestion
    /**
     * Make sure we have at least `nbytes` available for writing, allocates otherwise
     *
     * This is an inline-able fast path for `enlarge`. Prefer calling `enlarge` if the call
     * is known to cause allocation.
     */
```